lib/mifiel/document.rb

```
@@ -23,7 +23,7 @@ class Document < Mifiel::Base
     post :create_from_template, '/templates/:template_id/generate_document', timeout: 60
     post :create_many_from_template, '/templates/:template_id/generate_documents', timeout: 60
 
-    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable-next Metrics/MethodLength
```

RuboCop (1.89.0)

[warning] Malformed directive comment detected. The mode name must be one of `enable`, `disable`, `todo`, `push`, or `pop`.

(Lint/CopDirectiveSyntax)